posts/2022-07-12-components-with-dynamic-attributes.md

```diff
@@ -0,0 +1,272 @@
+%{
+  author: "Tracey Onim",
+  author_link: "https://github.com/TraceyOnim",
+  tags: ["LiveView", "Components", "LiveView Helpers"],
+  date: ~D[2022-07-12],
+  title: "Components with dynamic attributes",
+  excerpt: """
+  Learn how you can support dynamic attributes when using reusable components with assigns_to_attribute/2 function.
+  """
+}
+---
+
+
+I’m quite sure most of you, have already interacted with components you have created more components to reuse in your application.However, sometimes you may feel stuck when creating reusable components more so when you want components to meet certain criteria. For example, you may want the feature in component A to be different from that of component B.
```

Comment:
This statement is a little unclear:

> However, sometimes you may feel stuck when creating reusable components more so when you want components to meet certain criteria. For example, you may want the feature in component A to be different from that of component B.

Could you try re-phrasing it, even it that means you split up this explanation into multiple sentence or make it a bit longer? I'm not sure what you're trying to say here.